src/mpid/ch3/src/ch3u_port.c

```diff
@@ -990,7 +990,7 @@ int MPID_PG_BCast( MPIR_Comm *peercomm_p, MPIR_Comm *comm_p, int root )
     }
 
     /* Now, broadcast the number of local pgs */
-    mpi_errno = MPIR_Bcast( &n_local_pgs, 1, MPIR_INT_INTERNAL, root, comm_p, MPIR_ERR_NONE);
+    mpi_errno = MPIR_Bcast( &n_local_pgs, 1, MPIR_INT_INTERNAL, root, comm_p, 0);
```

Comment:
Obviously ch3 is a different but by the new guidelines these would be good candidations for the sync flag?

Reply:
Yes. Let me examine them and revise.